.gitignore

```diff
@@ -24,5 +24,4 @@ docs/fn_table
 docs/fn_to_op_table
 docs/op_autodoc
 docs/fn_autodoc
-docs/nvidia.ico
-dev/
\ No newline at end of file
+docs/nvidia.ico
```

Comment:
Please just remove this file from the PR.

Reply:
Done